eng/common/scripts/Publish-to-Npm.ps1

```diff
@@ -0,0 +1,190 @@
+param (
+  $pathToArtifacts,
+  $accessLevel,
+  $tag,
+  $additionalTag="",
+  $registry,
+  $npmToken,
+  $filterArg="",
+  $basicDeployment=$false,
+  $devopsFeed=$false,
+  $skipDiff=$false,
+  $packagesToPublishPath,
+  $addTag=$false
+)
+
+function replaceText($oldText,$newText,$filePath){
+    $content = Get-Content -Path $filePath -Raw
+    $newContent = $content -replace $oldText,$newText
+    if ($newContent -ne $content)
+    {
+        Set-Content -Path $filePath -Value $newContent -NoNewLine
+        Write-Host "replaceText [$oldText] [$newText] [$filePath]"
+    }
+}
+
+function updateContents($dirName, $version){
+    foreach ($md in $(dir $dirName -r -i *.md)){
+        replaceText "https://github.com/Azure/azure-sdk-for-js/tree/[^/]*" "" $md.Fullname
```

Comment:
Is this enough in a common context?

Reply:
yes, there's no dev release in tools repo now, if we need this for other repos in the future we can always add another if statement

Reply:
Dev is probably the biggest reason for this but the general reason for this is to ensure we don't upload an unchanged package. I agree we can potentially do this later but I also think we might be able to generalize this regex to make it work in the general case.